### PR TITLE
updating workflows - master

### PR DESCRIPTION
**Pull request by ATorrise**
Action | Description
-- | --
#1 | add [stale bot](https://github.com/zowe/zowe-cli/blob/master/.github/workflows/stale.yml)	
#2 | add [auto-comment](https://github.com/zowe/zowe-cli/blob/master/.github/workflows/auto-comment.yml) bot [on labels](https://github.com/zowe/zowe-cli/blob/master/.github/label-actions.yml)										
#3 | add issue template ([Bug](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), [Feature](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/feature_request.md))										
#4 | add [PR Template ](https://docs.google.com/document/u/0/d/1BvA8zNQs6QP1_gHJw7XsbEGNTTCH-v0SjM4kwMym3Cw/edit)   										
#5 | add [question redirect](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/config.yml)										
#6 | [update all workflows to remove deprecation warnings](https://github.com/zowe/zowe-cli-web-help-generator/pull/35/files), [octorelease updates](https://github.com/zowe/zowe-cli/pull/1712)										
#7 | remove Node 14 from workflow matrix

### CODE REVIEW

File: .github/workflows/changelog.yml
Comment (zFernand0) [inline .github/workflows/changelog.yml:1]: Can we also update `awharn/check_changelog_action@v0.0.2` to `v1`?

**Review by zFernand0 — CHANGES_REQUESTED** [review]
LGTM! 😋 
just one comment though 😅

**Review by t1m0thyj — APPROVED** [review]

File: .github/workflows/changelog.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - master
-      - zowe-v1-lts
```
Comment (zFernand0) [inline .github/workflows/changelog.yml:7]: there may be other workflows in this repo that could use the same update.
but I don't think it particularly matters for this branch or until we create a zowe-v2-lts branch

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -17,15 +17,15 @@ jobs:
...
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: 12.x
```
Comment (zFernand0) [inline .github/workflows/linter.yml:25]: we should probably use `lts/*` here
And please, don't forget to also update the step name 😋

**Review by zFernand0 — CHANGES_REQUESTED** [review]
LGTM, the only thing is that we have a node12 being setup for the linter.yml file
Can we update that to `lts/*`?

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**